### PR TITLE
Added super cool message from Zack

### PR DESCRIPTION
**Pull request by zackads**
I think this would significantly improve our codebase.  What do you reckon @blotty23?

### CODE REVIEW

**Comment by blotty23** [comment]
Yeah looks great! :D

File: foo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Zack woz ere
```
Comment (blotty23) [inline foo.txt:1]: I can't believe you wrote this!!

Comment (blotty23) [inline foo.txt:1]: `Emma woz ere`

Comment (zackads) [reply foo.txt:1]: Don't be ridiculous!

**Pull request merged**